### PR TITLE
[css-images-4][editorial] Add wpt annotations, fix #9800

### LINKED ISSUE #9800: [css-images-4] No WPT annotations; add them
Going through WPT which exercise the CSS Color 4 `<color-interpolation-method>`, I noticed that a bunch of these relate to CSS Images 4 and that specification has no WPT annotations, even though there are a lot of WPT (more than I had realized).

### PR DESCRIPTION
**Pull request by svgeesus**
image

Adds 3281 tests to the test results.

Note: inline test results are open in this PR, to make checking easier. Editors may well choose to set them to closed after merging this PR

@tabatkins @fantasai @LeaVerou

### CODE REVIEW

File: css-images-4/Overview.bs [hunk 1]
```diff
@@ -240,6 +242,47 @@ Resolution/Type Negotiation: the ''image-set()'' notation {#image-set-notation}
 		If not specified,
 		it has no effect on the <<image-set-option>>.
 
+	<wpt>
+		image-set/image-set-all-options-invalid.html
+	image-set/image-set-calc-x-rendering-2.html
```
Comment (LeaVerou) [inline css-images-4/Overview.bs:247]: Looks like there's a formatting issue here?

File: css-images-4/Overview.bs [hunk 2]
```diff
@@ -240,6 +242,47 @@ Resolution/Type Negotiation: the ''image-set()'' notation {#image-set-notation}
...
+	image-set/image-set-type-rendering-2.html
+	image-set/image-set-type-rendering-3.html
+	image-set/image-set-type-rendering.html
+	image-set/image-set-type-skip-unsupported-rendering.html
+	image-set/image-set-type-unsupported-rendering-2.html
+	image-set/image-set-type-unsupported-rendering.html
+	image-set/image-set-unordered-res-rendering.html
+	image-set/image-set-zero-resolution-rendering-2.html
+	image-set/image-set-zero-resolution-rendering.html
+	</wpt>
```
Comment (LeaVerou) [inline css-images-4/Overview.bs:285]:
```suggestion
	<wpt>
		image-set/image-set-all-options-invalid.html
		image-set/image-set-calc-x-rendering-2.html
		image-set/image-set-calc-x-rendering.html
		image-set/image-set-computed.sub.html
		image-set/image-set-conic-gradient-rendering.html
		image-set/image-set-content-rendering.html
		image-set/image-set-dpcm-rendering.html
		image-set/image-set-dpi-rendering-2.html
		image-set/image-set-dpi-rendering.html
		image-set/image-set-dppx-rendering.html
		image-set/image-set-empty-url-rendering.html
		image-set/image-set-first-match-rendering.html
		image-set/image-set-linear-gradient-rendering.html
		image-set/image-set-negative-resolution-rendering-2.html
		image-set/image-set-negative-resolution-rendering-3.html
		image-set/image-set-negative-resolution-rendering.html
		image-set/image-set-no-res-rendering-2.html
		image-set/image-set-no-res-rendering.html
		image-set/image-set-no-url-rendering.html
		image-set/image-set-parsing.html
		image-set/image-set-radial-gradient-rendering.html
		image-set/image-set-rendering-2.html
		image-set/image-set-rendering.html
		image-set/image-set-repeating-conic-gradient-rendering.html
		image-set/image-set-repeating-linear-gradient-rendering.html
		image-set/image-set-repeating-radial-gradient-rendering.html
		image-set/image-set-resolution-001.html
		image-set/image-set-resolution-002.html
		image-set/image-set-resolution-003.html
		image-set/image-set-type-first-match-rendering.html
		image-set/image-set-type-rendering-2.html
		image-set/image-set-type-rendering-3.html
		image-set/image-set-type-rendering.html
		image-set/image-set-type-skip-unsupported-rendering.html
		image-set/image-set-type-unsupported-rendering-2.html
		image-set/image-set-type-unsupported-rendering.html
		image-set/image-set-unordered-res-rendering.html
		image-set/image-set-zero-resolution-rendering-2.html
		image-set/image-set-zero-resolution-rendering.html
	</wpt>
```

**Review by LeaVerou — CHANGES_REQUESTED** [review]
LGTM, see minor formatting issue

**Review by LeaVerou — APPROVED** [review]

**Comment by tabatkins** [comment]
Sorry for not reviewing, was on vacation. Looks good, thanks for the review @LeaVerou!

**Pull request merged**